commerce_coordinator/apps/commercetools/tests/raw_ct_order.json

```diff
@@ -39,6 +39,10 @@
             "name": "brand-text",
             "value": "MichiganX"
           },
+          {
+            "name": "mode",
+            "value": "certified"
```

Comment:
Please use a value which is an actual course mode in edX/2U i.e. verified, professional etc.
This doesn't make a difference since its testing code but we should at least use values that concern our use case and logic.

Reply:
Done